themes/frontend-theme-valencia/src/css/_custom.scss

```diff
@@ -0,0 +1,492 @@
+	color: $color-theme-secondary;
+}
+
+.navbar.navbar-color-on-scroll li.active.btn a, .navbar.navbar-transparent li.active a {
+	color: #FFF;
+}
+
+.navbar.navbar-transparent ul.dropdown-menu li a {
+	color: #333;
+}
+
+@include sm {
+	.container > .navbar-header {
+		display: table-cell;
+		float: none;
+		vertical-align: middle;
+	}
+	.dropdown .dropdown-menu {
+		display: block;
+		visibility: hidden;
+		margin-top: -20px;
+		opacity: 0;
+		transition: all 150ms linear;
+	}
+
+	.dropdown.open .dropdown-menu {
+		visibility: visible;
+		margin-top: 1px;
+		opacity: 1;
+	}
+
+    .navbar-nav > li {
+        display: inline-block;
+    }
+	.navbar {
+		.navbar-collapse {
+			display: table-cell !important;
+			vertical-align: middle;
+			text-align: right;
+			width: 100%;			
+		}
+		&.navbar-transparent {
+			background-color: transparent;
+		}
+		.navbar-nav {
+        	> li {
+				display: inline-block;
+			}
+		}
+	}
+}
+
+@include md {
+	.navbar {
+		.navbar-center {
+			display: inline-block;
+		}
+		.navbar-brand {
+			height: 50px;
+			padding: 10px 15px;
+		}
+		.navbar-nav {
+			> li {
+				margin: 0;
+				&.btn {
+					margin: 0 10px;
+					&:last-child {
+						margin-right: 0;
+					}
+				}
+			}
+		}
+	}
+}
+
+/* ---------- Typography ---------- */
+
+h1, h2, h3, h4, h5, h6 {
+    line-height: 1.5em;
+}
+
+h1, h3 {
+    font-weight: 700;
+	font-family: $font-serif;
+    color: $color-font-header;
+}
+
+h2, h4, h6 {
+    font-weight: 700;
+	font-family: $font-sans;
+    color: $color-font-header;
+}
+
+h1 {
+	font-size: 3.8em;
+	line-height: 1.15em;
+}
+
+h2 {
+	font-size: 2.6em;
+}
+
+h3 {
+	font-size: 1.825em;
+}
+
+h4 {
+	font-size: 1.25em;
+	line-height: 1.55em;
+}
+
+h5 {
+	font: 300 1.25em $font-sans;
+	line-height: 1.55em;
+    color: #777;
+}
+
+h6 {
+	font-size: 0.9em;
+	font-weight: 500;
+	text-transform: uppercase;
+}
+
+p {
+	color: $color-font-body;
+    font-size: 1.142em;
+    margin-bottom: 10px;
+   
+}
+
+.light-text {
+    color: #FFF !important;
+    h1, h2, h3, h4, h5, h6, p {
+        color: #FFF !important;
+    }
+}
+
+.site-title {
+	font: 700 4.2em $font-sans;
+    line-height: 1.15em;
+    color: #FFF;
+}
+
+.carousel h2 {
+	color: #FFF;
+    font: 300 1.3em $font-serif;
+    line-height: 1.4em;
+    margin-bottom: 30px;
+}
+
+@include md {
+	.carousel h2 {
+		margin-bottom: 60px;
+	}
+}
+
+@include lg {
+	.site-title {
+		font-size: 4.8em;
+	}
+}
+
+/* ---------- Buttons & Links ---------- */
+
+a {
+	color: $color-theme-main;
+	transition: all .3s ease;
+}
+
+a:focus, a:hover {
+	color: $color-theme-secondary;
+	text-decoration: none;
+}
+
+.btn, input[type="submit"] {
+  position: relative;
+  margin: 10px 1px;
+  padding: 12px 30px;
+  border: none;
+  border-radius: 3px;
+  font-size: 12px;
+  text-transform: uppercase;
+  transition: box-shadow 0.2s cubic-bezier(0.4, 0, 1, 1), 
+              background-color 0.2s cubic-bezier(0.4, 0, 0.2, 1);
+  will-change: box-shadow,transform; 
```

Comment:
Your indentations are different for this block vs the other blocks.  Probably an issue with spaces and tabs.